cli/features/tutorial.feature

```diff
@@ -134,7 +134,9 @@ Feature: Following the tutorial
           - 'mongo'
       """
     When running "exo setup" in this application's directory
+    And setting up a temporary mongo docker container
```

Comment:
In my understanding, `exo test` should start and stop the Mongo Docker container. Because it sees that a service needs MongoDB when looking through the `service.yml` files.